AST/implementation/implementation.dart

```diff
@@ -651,11 +651,8 @@ class Implementation {
   }
 
   static String processPrefixedIdentifier(PrefixedIdentifier identifier) {
-    var csharp = "";
-    for (var entity in identifier.childEntities) {
-      csharp += processEntity(entity);
-    }
-    return csharp;
+    var last = identifier.childEntities.last.toString();
+    return last;
```

Comment:
Can't do this, because it then breaks lots of other things. If this is purely for the `this.value` parameter type, then it might be best to just make an exception like, if parameterName.Contains('this.') remove it.